### DISCUSSION TITLE
Release Update for werkzeug Version 2.3.9

### DISCUSSION THREAD

**Discussion by Fares-Abubaker** [pallets/werkzeug#2898] [Q&A] [answered]
Hi,

With the recent vulnerability `CVE-2024-34069` in `werkzeug`, is there a plan to release version 2.3.9 to address this issue? Or should we start transitioning to version 3?

Thanks.

**Comment by ThiefMaster** [comment, marked as answer]
You need to update to a supported version (ie werkzeug 3).

In any case, this CVE is only in the debugger which must ONLY be used in development anyway. So for a normal deployment you can safely ignore it.

**Reply by Fares-Abubaker** [reply]
Many thanks
